### ISSUE TITLE
'clocker data' shows incorrect totals

### ISSUE THREAD

**Issue by noffle** [substack/clocker#33]
Both days have incorrect totals, off by several hours.

```
% clocker list
1454519171  2016-02-03  [ 09:06:11 - 10:14:31 ]  (01:08:20)
1454538228  2016-02-03  [ 14:23:48 - 15:33:06 ]  (01:09:18)
1454612157  2016-02-04  [ 10:55:57 - 11:21:05 ]  (00:25:08)
1454616412  2016-02-04  [ 12:06:52 - 17:19:36 ]  (05:12:44)

% clocker data
[
  {
    "hours": [
      {
        "date": "2016-02-03",
        "hours": 4.59
      },
      {
        "date": "2016-02-04",
        "hours": 11.26
      }
    ],
    "title": "consulting"
  }
]
```

**Comment by fnogatz** [comment]
Thank you for your bug report! Never experienced this bug as I always set a type :) Fixed via 14286bf by adding missing `return` statements. Published as new version 1.11.2 on npm.

**Comment by noffle** [comment]
Thanks @fnogatz!
